tasks/60_convert-html-entities/convert-html-entities.js

```diff
@@ -0,0 +1,21 @@
+// For in loop with String.replace method.
+const entities = [
+  {entity: '&amp;',
+    sign: /&/g},
+  {entity: '&lt;',
+    sign: /</g},
+  {entity: '&gt;',
+    sign: />/g},
+  {entity: '&apos;',
+    sign: /'/g},
+  {entity: '&quot;',
+    sign: /"/g},
+];
+
+export const convertHTML = ( str ) => {
+  // eslint-disable-next-line guard-for-in
+  for ( let key in entities) {
```

Comment:
`for of` or `forEach` will looks better, use `for in` only for iterating object properties but not arrays

also you can have solve it with using map(or simple object) where as key you will have sign and entity as value of this key, also here you can pass to replace function and just take symbol which was matched from your map